.github/workflows/test_frameworks.yml

```diff
@@ -186,7 +186,8 @@ jobs:
         env:
           # Disabled distributed tracing since there are a lot of tests that assert on headers
           DD_HTTPX_DISTRIBUTED_TRACING: "false"
-        run: pytest
+        # test_pool_timeout raises RuntimeError: The connection pool was closed while 1 HTTP requests/responses were still in-flight
```

Comment:
is this error because of us or exists upstream too?

Reply:
This exists upstream. I ran this action without install ddtrace and I got the same errror. 

Test Action:
```
 httpx-testsuite-0_22_0:
    runs-on: ubuntu-latest
    defaults:
      run:
        working-directory: httpx
    steps:
      - uses: actions/checkout@v2
        with:
          repository: encode/httpx
          ref: 0.22.0
          path: httpx
      - uses: actions/setup-python@v2
        with:
          python-version: '3.9'
      - name: Install dependencies
        run: pip install -r requirements.txt
      - name: Run tests
        env:
          # Disabled distributed tracing since there are a lot of tests that assert on headers
          DD_HTTPX_DISTRIBUTED_TRACING: "false"
        run: pytest
```

Error: 
```
 self = <httpcore.AsyncConnectionPool object at 0x7f83f140fa30>
| 
|     async def aclose(self) -> None:
|         """
|         Close any connections in the pool.
|         """
|         async with self._pool_lock:
|             requests_still_in_flight = len(self._requests)
|     
|             for connection in self._pool:
|                 await connection.aclose()
|             self._pool = []
|             self._requests = []
|     
|             if requests_still_in_flight:
| >               raise RuntimeError(
|                     f"The connection pool was closed while {requests_still_in_flight} "
|                     f"HTTP requests/responses were still in-flight."
|                 )
| E               RuntimeError: The connection pool was closed while 1 HTTP requests/responses were still in-flight.
| 
| /opt/hostedtoolcache/Python/3.9.10/x64/lib/python3.9/site-packages/httpcore/_async/connection_pool.py:303: RuntimeError
| ----------------------------- Captured stdout call -----------------------------
| INFO:     127.0.0.1:65216 - "GET / HTTP/1.1" 200 OK
| ======================== 2 failed, 674 passed in 6.55s =========================
```

Reply:
nice, thanks for checking!